integrations/clickup/README.md

```diff
@@ -0,0 +1,7 @@
+# ClickUp
+
+An integration used to import ClickUp resources into Port.
+
+#### Install & use the integration - [Integration documentation](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/) *Replace this link with a link to this integration's documentation*
```

Comment:
once we release this to production and finalize the docs, the link will be https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/project-management/clickup/ so update it here

Reply:
Updated url to match the one when we finalize the docs